mdoc/Test/en.expected-eii-implementation-ecmadoc/CustomNamespace/ClassEnumerator.xml

```diff
@@ -0,0 +1,116 @@
+  <TypeSignature Language="VB.NET" Value="Public NotInheritable Class ClassEnumerator&#xA;Implements CustomInterface" />
+  <TypeSignature Language="F#" Value="type ClassEnumerator = class&#xA;    interface CustomInterface" />
+  <TypeSignature Language="JavaScript" Value="function ClassEnumerator()" Usage="var classEnumerator = new ClassEnumerator();" />
+  <TypeSignature Language="C++ CLI" Value="public ref class ClassEnumerator sealed : CustomNamespace::CustomInterface" />
+  <TypeSignature Language="C++ CX" Value="public ref class ClassEnumerator sealed : CustomNamespace::CustomInterface" />
+  <TypeSignature Language="C++ WINRT" Value="class ClassEnumerator sealed : CustomNamespace::CustomInterface" />
+  <AssemblyInfo>
+    <AssemblyName>DocTest-VB-Eii</AssemblyName>
+    <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+  </AssemblyInfo>
+  <Base>
+    <BaseTypeName>System.Object</BaseTypeName>
+  </Base>
+  <Interfaces>
+    <Interface>
+      <InterfaceName>CustomNamespace.CustomInterface</InterfaceName>
+    </Interface>
+  </Interfaces>
+  <Docs>
+    <summary>ECMA Type desc summary</summary>
+    <remarks>ECMA Type desc remarks</remarks>
+  </Docs>
+  <Members>
+    <Member MemberName=".ctor">
+      <MemberSignature Language="C#" Value="public ClassEnumerator ();" />
+      <MemberSignature Language="C++ CLI" Value="public:&#xA; ClassEnumerator();" />
+      <MemberSignature Language="C++ CX" Value="public:&#xA; ClassEnumerator();" />
+      <MemberSignature Language="C++ WINRT" Value=" ClassEnumerator();" />
+      <MemberSignature Language="ILAsm" Value=".method public specialname rtspecialname instance void .ctor() cil managed" />
+      <MemberSignature Language="JavaScript" Value="function ClassEnumerator()" Usage="var classEnumerator = new ClassEnumerator();" />
+      <MemberSignature Language="VB.NET" Value="Public Sub New ()" />
+      <MemberType>Constructor</MemberType>
+      <AssemblyInfo>
+        <AssemblyVersion>0.0.65535.65535</AssemblyVersion>
+      </AssemblyInfo>
+      <Parameters />
+      <Docs>
+        <summary>ECMA ctor summary</summary>
+        <remarks>ECMA ctor remarks</remarks>
+      </Docs>
+    </Member>
+    <Member MemberName="CustomNamespace.CustomInterface.Prop1" ExplicitInterfaceMemberName="CustomProp1">
+      <MemberSignature Language="C#" Value="int CustomNamespace.CustomInterface.Prop1 { get; set; }" />
+      <MemberSignature Language="C++ CLI" Value="property int CustomNamespace::CustomInterface::Prop1 { int get(); void set(int value); };" />
+      <MemberSignature Language="C++ CX" Value="property int CustomNamespace::CustomInterface::Prop1 { int get(); void set(int value); };" />
+      <MemberSignature Language="F#" Value="" Usage="CustomNamespace.CustomInterface.Prop1" />
+      <MemberSignature Language="ILAsm" Value=".property instance int32 CustomProp1" />
+      <MemberSignature Language="VB.NET" Value=" Property Prop1 As Integer Implements CustomInterface.Prop1" />
```

Comment:
Shouldn't the VB.NET signature here be `Property CustomProp1 As Integer Implements CustomInterface.Prop1`? It shows `ExplicitInterfaceMemberName="CustomProp1"`